vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -3,8 +3,9 @@ sidebarDepth: 2
 ---
 # Manage NGINX
 
+This page assumes you are running Entando on [Amazon Elastic Kubernetes Service](../getting-started/eks-install.md), [Azure Kubernetes Service](../getting-started/azure-install.md) or [Google Kubernetes Engine](../getting-started/gke-install.md) and have already replaced the default ingress controller with NGINX. The following sections explain how to verify and refine your NGINX configuration.
```

Comment:
Yeah, I agree. I said prereq in my last comment and it's a little more general than that.  Maybe this:

There are environments where it's useful to use NGINX rather than the default ingress controller. This page shows how to verify and refine your NGINX configuration. 

See the following install guides if needed:
* [Amazon Elastic Kubernetes Service (EKS)](../../tutorials/getting-started/eks-install.md#install-the-nginx-ingress-controller)
* [Azure Kubernetes Service (AKS)](../../tutorials/getting-started/azure-install.md#deploy-nginx-ingress-controller)
* [Google Kubernetes Engine (GKE)](../../tutorials/getting-started/gke-install.md#install-the-nginx-ingress-controller)

---
Note: the links weren't quite right in the previous version.